lua/user/plugins.lua

```diff
@@ -527,130 +519,133 @@ M.config = function()
     },
     {
       "saecki/crates.nvim",
       event = { "BufRead Cargo.toml" },
-      requires = { { "nvim-lua/plenary.nvim" } },
+      dependencies = { { "nvim-lua/plenary.nvim" } },
       config = function()
         require("user.crates").config()
       end,
-      disable = not lvim.builtin.rust_programming.active,
+      enabled = lvim.builtin.rust_programming.active,
     },
     {
       "hrsh7th/cmp-cmdline",
-      disable = not lvim.builtin.fancy_wild_menu.active,
+      enabled = lvim.builtin.fancy_wild_menu.active,
     },
     {
       "mrjones2014/legendary.nvim",
       config = function()
         require("user.legendary").config()
       end,
-      disable = not lvim.builtin.legendary.active,
+      enabled = lvim.builtin.legendary.active,
     },
     {
       "stevearc/dressing.nvim",
+      lazy = true,
       config = function()
         require("user.dress").config()
       end,
-      disable = not lvim.builtin.dressing.active,
+      enabled = lvim.builtin.dressing.active,
       event = "BufWinEnter",
     },
     {
       "kdheepak/cmp-latex-symbols",
-      requires = "hrsh7th/nvim-cmp",
+      dependencies = "hrsh7th/nvim-cmp",
       ft = "tex",
     },
     {
       "ThePrimeagen/refactoring.nvim",
+      lazy = true,
       ft = { "typescript", "javascript", "lua", "c", "cpp", "go", "python", "java", "php" },
       event = "BufRead",
       config = function()
         require("refactoring").setup {}
       end,
-      disable = not lvim.builtin.refactoring.active,
+      enabled = lvim.builtin.refactoring.active,
     },
     {
       "b0o/incline.nvim",
       config = function()
         require("user.incline").config()
       end,
-      disable = lvim.builtin.winbar_provider ~= "filename",
+      enabled = lvim.builtin.winbar_provider == "filename",
     },
     {
       "fgheng/winbar.nvim",
       config = function()
         require("user.winb").config()
       end,
       event = { "InsertEnter", "CursorHoldI" },
-      disable = lvim.builtin.winbar_provider ~= "treesitter",
+      enabled = lvim.builtin.winbar_provider == "treesitter",
     },
     {
       "SmiteshP/nvim-gps",
       module_pattern = { "gps", "nvim-gps" },
       config = function()
         require("user.gps").config()
       end,
-      requires = "nvim-treesitter/nvim-treesitter",
+      dependencies = "nvim-treesitter/nvim-treesitter",
       event = { "InsertEnter", "CursorHoldI" },
-      disable = lvim.builtin.winbar_provider ~= "treesitter",
+      enable = lvim.builtin.winbar_provider == "treesitter",
     },
     {
       "vimpostor/vim-tpipeline",
-      disable = not lvim.builtin.tmux_lualine,
+      enabled = lvim.builtin.tmux_lualine,
     },
     {
       "stevearc/overseer.nvim",
       config = function()
         require("user.ovs").config()
       end,
-      disable = lvim.builtin.task_runner ~= "overseer",
+      enable = lvim.builtin.task_runner == "overseer",
```

Comment:
Shouldn't this be `enabled` instead of `enable` ?

Reply:
Fixed in f090f8a